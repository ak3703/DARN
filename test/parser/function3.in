int main(bool y) { 
	int y;
	y = 4;
	1 + 2; 
}