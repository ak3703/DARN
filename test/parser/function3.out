
fdecl { 
int main(y) {
vdecl { 
int id y;
}
Assign { var y =  int_lit 4 };
Binop { int_lit 1 + int_lit 2 };
} 
}
